pymeasure/instruments/keithley/keithley2260B.py

```diff
@@ -0,0 +1,71 @@
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import (
+    # truncated_range,
+    # truncated_discrete_set,
+    strict_discrete_set,
+)
+# from pymeasure.adapters import VISAAdapter
+from .buffer import KeithleyBuffer
+
+import logging
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class Keithley2260B(Instrument):
+    """ Represents the Keithley 2260B Power Supply (minimal implementation)
+    and provides a high-level interface for interacting with the instrument.
+
+    For a connection through tcpip, the socket is hardcoded to 2268,
+        example connection string: 'TCPIP::xxx.xxx.xxx.xxx::2268::SOCKET'
+        the read termination is \n
+
+    .. code-block:: python
+
+        source = Keithley2260B("GPIB::1")
+        source.voltage = 1
+        print(source.voltage)
+        print(source.current)
+        print(source.power)
+        print(source.applied)
+
+    """
+
+    enabled = Instrument.control(
+        "OUTPut?", "OUTPut %d",
+        """A boolean property that controls whether the source is enabled, takes
+        values True or False (1 or 0). """,
+        validator=strict_discrete_set,
+        values={True: 1, False: 0},
+        map_values=True
+    )
+
+    current = Instrument.control(
+        ":SOUR:CURR?", ":SOUR:CURR %g",
+        """ A floating point property that controls the source current
+        in amps. The allowed range is not fixed. """,
```

Comment:
Maybe better wording: "This is not checked against the allowed range"

Reply:
When reading, does this return the actual current achieved, or the set current limit?

Reply:
I think, this reads the set current limit - however, the device has the capability to operate both in constant voltage (CV) and constant current (CC) modes, and switches between the two depending on the set values and the applied load. if values are set which are outside the boundaries given by these circumstances, the instrument will ignore them (as far as I understand the manual). 
Depending on the mode (CC or CV), the reading gives either the current/voltage limit, or the actual value achieved, which is not immediately clear without further communication with the instrument. 

For a "full" implementation of this device to pymeasure, I guess it would be good to correctly observe these modes and read values for limits and actual achieved values separately (if possible). The implementation I can provide will necessarily lack this ability, and will only include the most prominent features of (most any keithley) power supplies, where some device-specific preferences and the convenience control `applied` are added.

Reply:
OK, I suspected that, but it was not fully clear from the manual, and I don't have the device around to check.

When I implement power supplies (that have the necessary commands), I tend to define a quartet of `control`s of `voltage_setpoint` (the one you set), `voltage` (the actual voltage the device achieves, which can deviate in CC mode), `current` (the actual current, less than the limit in CV mode) and `current_limit` (the limit you set).
I found this to be the clearest way to express what is going on (and enabling expressive interaction/logging), maybe you also find that useful.

Reply:
Thank you for your suggestions! I have made changes accordingly.